github3/repos/repo.py

```diff
@@ -1282,6 +1282,32 @@ def imported_issue(self, imported_issue_id):
         return self._instance_or_null(ImportedIssue, json)
 
     @requires_auth
+    def imported_issues(self, number=-1, since=None, etag=None):
+        """Retrieve imported issues
+
+        :param int number: (optional), number of imported issues to return.
+            Default: -1 returns all branches
+        :param since: (optional), Only commits after this date will
```

Comment:
> So if this needs to be ISO8601, I think we have a helper to ensure that it is and we should probably be using it here. Your unit test is kind of wrong because GitHub expects a specific format of ISO8601 (even though the date you pass there is technically valid ISO8601 iirc).

@sigmavirus24 , per the gist (https://gist.github.com/jonmagic/5282384165e0f86ef105#request-3), `since` parameter accepts "a date or a date and time".  In their example, they use:

https://api.github.com/repos/#{GITHUB_USERNAME}/foo/import/issues?since=2015-03-15

Do we want to enforce the user of ISO8601/datetime object for consistency - regardless of what the endpoint supports?  Would this prevent them from sending just `date` (i.e 2015-03-15 ) ?